### ISSUE TITLE
You're calling logging in Facebook with an activity supports androidx activity result APIs. Please follow our document to upgrade to new APIs to avoid overriding onActivityResult().

### ISSUE THREAD

**Issue by ashiq-mvp** [Innim/flutter_login_facebook#107]
Description:
The Facebook login app must be upgraded to the latest version (v12.0).

Attachment:
![296179282-2e7654f3-ae31-46e7-aa81-872a3979a964](https://github.com/Innim/flutter_login_facebook/assets/133241376/103ab9e4-e73f-4b30-9170-92af24f616a6)
<img width="995" alt="296182676-1832ade0-0707-4394-8421-9198b5c95b0e" src="https://github.com/Innim/flutter_login_facebook/assets/133241376/80b0c88c-ef99-425f-ba35-0bc2646be020">

**Comment by greymag** [comment]
Hey. Do you use the latest version of the plugin? Plugin use Facebook SDK 16.2.0

**Comment by greymag** [comment]
Then you should be fine. I don't know why Facebook shows this old versions,  I'm using this plugin in several apps and see different SDK versions for different apps, but all of them are greater than required.

![Screenshot 2024-02-20 at 14 19 29](https://github.com/Innim/flutter_login_facebook/assets/1502131/5fd45f95-d417-4c39-96c2-a7a8c6df2eb0)

Maybe your Facebook app is receiving requests from old APKs and that's the reason you're seeing this warning. 

Bottom line I think the integration is OK and there shouldn't be any problems. But please let me know if you figure out why Facebook is showing an invalid SDK version

**Comment by ashiq-mvp** [comment]
As you mentioned the Package is working fine (Note: in release APP). I'm still unsure why I received this message from Facebook, but I appreciate your support.
